django/db/backends/mysql/creation.py

```diff
@@ -75,7 +75,7 @@ def _clone_db(self, source_database_name, target_database_name):
         load_cmd[-1] = target_database_name
 
         with subprocess.Popen(
-            dump_cmd, stdout=subprocess.PIPE, env=dump_env
+            dump_cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE, env=dump_env
```

Comment:
I’m not sure there’s a need to capture and then re-output the stderr. It will already appear on screen by default, no?

Reply:
It would, but my thinking is that it was better to give some context about the failure *before* dumping the error message. That's how things are normally done, right?

Reply:
My concern is that if we always capture stderr, but only sometimes display it, any warnings that `mysqldump` would show will be masked. I think it would be better to just let it pass through. If we log an error and exit, then users should be able to read the preceeding output from mysqldump.

Reply:
I've started logging this again but with a preceding context comment. Let me know what you think.